web/src/components/ui/calendar-range.tsx

```diff
@@ -369,7 +369,9 @@ export function DateRangePicker({
                 }}
               >
                 <SelectTrigger className="mx-auto mb-2 w-[180px]">
-                  <SelectValue placeholder="Select..." />
+                  <SelectValue
+                    placeholder={`${t("export.select", { ns: "components/dialog" })}...`}
```

Comment:
Probably want to include the ellipsis as part of the key here

Reply:
it doesn't fit in the other area that it is used, the export dialog

Reply:
I'd just make a separate key then, since I'm not sure the ellipsis will translate correctly into other languages, especially RTL languages